Refactored class

joshmtucker opened the pull request:
• Perspective is a class (doesn’t extend Layer)
• Variables for Device.screen Device.phone
• Functions are part of the class

NOTE - Missing pan events (need to create new way / create a handling outside of device)

Commit:
Refactored class

• Perspective is a class (doesn’t extend layer)
• Variables for Device.screen Device.phone
• Functions are part of the class

NOTE - Missing events

mrrocks commented on perspective-view.framer/framer/backups/backup-1460224356.coffee:
```diff
@@ -1,8 +1,8 @@
+{Perspective} = require "perspective-view"
 
-myPerspective = new PerspectiveLayer
+p = new Perspective
```
Can we use a semantic variable name?

joshmtucker replied:
Yep - committed new version with class "PerspectiveView" and variable name "perspectiveView".

Commit:
Added event handling

When deviceType is “fullscreen” then the screen is pawnable. Otherwise
the device frame is pawnable.

Commit:
Changed class to "PerspectiveView"

Also added semantic variable name.
